core/src/main/java/de/danoeh/antennapod/core/ClientConfig.java

```diff
@@ -41,6 +42,7 @@ public static synchronized void initialize(Context context) {
         UpdateManager.init(context);
         PlaybackPreferences.init(context);
         NetworkUtils.init(context);
+        CastManager.init(context);
```

Comment:
This always gets called regardless of user preferences, which may or may not give problems if the user doesn't have play services installed. So we need to test for that possibility.
Ideally, our app shouldn't require play services in order to run, and users who don't have it won't be able to even activate cast support in the options.